src/include/daos_srv/smd.h

```diff
@@ -40,11 +40,14 @@ struct smd_pool_info {
 	d_list_t	 spi_link;
 	uuid_t		 spi_id;
 	uint64_t	 spi_blob_sz[SMD_DEV_TYPE_MAX];
-	uint32_t	 spi_tgt_cnt[SMD_DEV_TYPE_MAX];
+	uint16_t	 spi_flags[SMD_DEV_TYPE_MAX];
+	uint16_t	 spi_tgt_cnt[SMD_DEV_TYPE_MAX];
```

Comment:
this will break interoperability right? I don't think we can change it. If you really need to add status for each target, I'm afraid you have to create another table.

Reply:
ah, nm, this is a data structure in DRAM, sorry for the noise